### ISSUE TITLE
Contrast ratio for Best Bets link fails

### ISSUE THREAD

**Issue by karlikpj** [NCIOCPL/sitewide-search-app#88]
### Issue description

```
.title-and-desc > .title
<a class="title" href="/about-cancer/understanding/what-is-cancer">What Is Cancer?</a>
```

Element has insufficient color contrast of 3.88 
(foreground color: #2b7bba, background color: #dbf2f8, font size: 12.0pt (16px), font weight: normal). 

Expected contrast ratio of 4.5:1

> **ESTIMATE** 20

### Steps to reproduce the issue

1.  Go to https://react-app-dev.cancer.gov/sitewide-search-app/develop/?cfg=0&swkeyword=tumor
2. inspect colors between background of best bets and link (use Wave or other a11y inspector)
3. See it fails..

### What's the expected result?
Expected contrast ratio of 4.5:1

### What's the actual result?
Element has insufficient color contrast of 3.88 

### Additional details / screenshot
- ![Screenshot]()

### Related Tickets
* Issue #9999
* Issue #9999
